Source/Diagnostics/ReducedDiags/ParticleHistogram.cpp

```diff
@@ -152,8 +152,13 @@ void ParticleHistogram::ComputeDiags (int step)
         {
             auto const w  = p.rdata(PIdx::w);
             auto const x  = p.pos(0);
+#if AMREX_SPACEDIM == 3
```

Comment:
We can probably use `GetParticlePosition` here, I am just unsure what happens downstream in `fun_partparser`.

Reply:
I think we to handle RZ here would be to use the `GetParticlePosition` functor, but unpack the result here and set `y` to `0` before handing to the `fun_partparser`.

Reply:
I think for RZ we would keep the values, just for 2D we would reset `y` to zero.
Although we need to test if that's needed and `y` is even accessed.